### PR TITLE
Add Consul ServiceID to the discovery meta labels.

### PR DESCRIPTION
**Pull request by mrwacky42**
@brian-brazil  - Check it out

### CODE REVIEW

**Comment by fabxc** [comment]
:+1:

File: retrieval/discovery/consul.go [hunk 1]
```diff
@@ -276,6 +278,7 @@ func (cd *ConsulDiscovery) watchService(srv *consulService, ch chan<- *config.Ta
 				ConsulTagsLabel:           clientmodel.LabelValue(tags),
 				ConsulServiceAddressLabel: clientmodel.LabelValue(node.ServiceAddress),
 				ConsulServicePortLabel:    clientmodel.LabelValue(strconv.Itoa(node.ServicePort)),
+				ConsulServiceIDLabel:      clientmodel.LabelValue(node.ServiceID),
```
Comment (brian-brazil) [inline retrieval/discovery/consul.go]: Is there anything else in node that we may want to add?

Comment (mrwacky42) [inline retrieval/discovery/consul.go]: I think ```ServiceID``` was the only item that was excluded.
See also:
https://github.com/hashicorp/consul/blob/c8f76b5e2cf6bfdc52af1924114e8dff40394821/website/source/docs/agent/http/catalog.html.markdown#-v1catalogserviceservice

Comment (brian-brazil) [inline retrieval/discovery/consul.go]: Great.

**Pull request merged**